public/fonts/fonts-desktop.css

```diff
@@ -0,0 +1,72 @@
+@font-face {
+  font-family: 'Merriweather';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Merriweather'), url(http://localhost:41050/calypso/fonts/Merriweather.ttf) format('truetype');
```

Comment:
Not sure what you mean here? Once we move loading of CSS assets to webpack loaders we'll likely want all this under a common.css file.